src/components/app/deviceGroupDropdown/deviceGroupDropdown.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+
+.device-group-dropdown {
+  @include rem-fallback(width, 200px);
+  margin-right: auto; /* Push to the right of the other context menu buttons*/
+
+  .Select-menu-outer {
```

Comment:
what is our naming convention for scss with upper-case vs lower-case. I see `.device-group-dropdown` vs `.Select-menu-outer`

Reply:
Our is all lowercase. However, in this case I'm overriding styles inside the react-select component, which follows a different css class naming pattern.